requirements.txt

```diff
@@ -38,3 +38,7 @@ nbval
 sphinx
 sphinx-gallery
 pydata-sphinx-theme
+
+bokeh
```

Comment:
This looks like it'll need an update after the split

Reply:
Good point....

Reply:
Done